packages/web-sdk/src/utils/webStorage.ts

```diff
@@ -25,7 +25,8 @@ export function isWebStorageAvailable(type: StorageMechanism): boolean {
     return true;
   } catch (error) {
     // the above can throw
-    faro.internalLogger?.info(`Web storage of type ${type} is not available. Reason: ${error}`);
+    // this is called during module init, when the global instance may not be available
```

Comment:
This is not necessarily called during global module init if you would be able to rephrase the comment for clarity.